eksperyment_mikroserwisy.rst

```diff
@@ -0,0 +1,78 @@
+﻿Testowanie mikrousług/Dockera
```

Comment:
Osobiście jestem zwolennikiem nieograniczania długości linij. Dostosowywanie się do terminali 80-znakowych współcześnie nie wydaje się mieć podstaw, zresztą nawet im nikt nie broni łamać linij, gdy są za długie. Git potrafi pokazywać różnice na poziomie poszczególnych słów (`git diff --word-diff`), zresztą nawet Github to potrafi (choć nie w trybie komentowania, tylko wyświetlania).

No ale temat jest kontrowersyjny.